packages/eightballer/connections/dcxt/interfaces/asset_bridging.py

```diff
@@ -95,6 +104,19 @@ async def request_bridge(  # noqa: PLR0911
                 amount=amount,
                 receiver=receiver,
             )
+            if tx_result is None:
+                return reply_err(
+                    code=ErrorCode.Code.CODE_OTHER_EXCEPTION,
+                    err_msg="Transaction failed to be submitted.",
+                )
```

Comment:
`tx_result` can never be `None`: [source](https://github.com/8ball030/derive_client/blob/6495d81dc0fc21d1bd7d3cbc76e29e40825420a9/derive_client/clients/base_client.py#L162-L184)